docs/modules/operation/pages/deep-dive/events/sources/syslog.adoc

```diff
@@ -63,24 +63,24 @@ Arrange the patterns in the file from most specific to least specific, since the
 | String containing only valid IP address characters (IPv4 or IPv6).
 
 | MONTH
-| Three-character English month abbreviation
+| Three-character English abbreviation of the month.
 
 | NOSPACE
 | String that contains no whitespace.
 
 | STRING
 | String.
-Because this matches any character, it must be followed by a delimiter in the pattern String.
+Because this matches any character, it must be followed by a delimiter in the pattern string.
 
 | WHITESPACE
-| String that contains only whitespace (spaces and or tabs).
+| String that contains only whitespace (spaces and tabs).
 |===
 
-[options="header"]
+[caption=]
+.`RadixTreeSyslogParser` semantic tokens
```

Comment:
This is not showing up as a heading/label in the built file.

Reply:
Hmmm... maybe that's because of the unicode formatting? I'll do some troubleshooting